jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -328,6 +330,37 @@ public void close() {
     }
   }
 
+  private boolean notExistAccountInProperty() {
```

Comment:
maybe to change this function to `existAccountInProperty` and return `property.containsKey("default.user") && property.containsKey("default.password")` and change to `!existAccountInProperty()` where we call it in `setAccountOfCredential`, just suggestion for simplicity :)